src/misc/cache/lv_image_cache.h

```diff
@@ -33,13 +33,27 @@ extern "C" {
  */
 void lv_image_cache_drop(const void * src);
 
+/**
+ * Resize image cache.
+ * @param new_size  new size of the cache in bytes or count of images..
```

Comment:
Is "count of images" still relevant?
I thought only the size (bytes) matters.

Reply:
You're right, image cache uses size in bytes but header in count